internal/processor/otel/traces_test.go

```diff
@@ -1341,6 +1341,20 @@ func TestServiceTarget(t *testing.T) {
 	})
 }
 
+func TestGRPCTransactionFromNodejsSDK(t *testing.T) {
```

Comment:
Can you please also add a test (maybe a subtest of this one) that creates a gRPC client span, and verify that the `span.type` and `span.subtype` are "external" and "grpc" respectively?

Reply:
👍 Makes sense, tackling this today.

Reply:
@axw Change made, thanks for catching this one.